site/en/docs/web-platform/launch-handler/index.md

```diff
@@ -27,80 +31,69 @@ The Launch Handler API lets you control how your app is launched, for example, w
 
 </div>
 
-{% Aside %} 
 ### Try out the Launch Handler API
 
 During the trial phase you can test the API by one of two methods.
 
 #### Local testing
 
-To experiment with the Launch Handler API locally, without an origin trial token, enable the `#enable-desktop-pwas-launch-handler` flag in `about://flags`.
+To experiment with the Launch Handler API locally, without an origin trial token, enable the
+`#enable-desktop-pwas-launch-handler` flag in `about://flags`.
 
 #### Register for the origin trial
 
-Starting in Chromium 98, the Launch Handler API will be available as an [origin trial](/docs/web-platform/origin-trials/) in Chromium. The origin trial is expected to end in Chromium 102 (June 15, 2022). [Register here](https://developers.chrome.com/origintrials/#/trials/active).
+Starting in Chromium 98, the Launch Handler API will be available as an
+[origin trial](/docs/web-platform/origin-trials/) in Chromium. The origin trial is expected to end
+in Chromium 106 (October 26, 2022).
```

Comment:
```suggestion
in Chromium 106 (October 25, 2022).
```

Reply:
This is the date [on the dashboard](https://developer.chrome.com/origintrials/#/view_trial/2978005253598740481).